tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,182 @@
+		const deleteContainer = contactButtonActionsContainer.locator( '.elementor-editor-element-remove' );
+		await expect( deleteContainer ).toBeVisible();
+		await deleteContainer.click();
+		const libraryModal = page.locator( '#elementor-template-library-modal' );
+		await expect( libraryModal ).toBeVisible();
+
+		await floatingElPage.goToFloatingButtonsPage();
+		const floatingElementTitle = page.locator( '.wp-list-table tbody tr td.page-title a.row-title' );
+		await expect( floatingElementTitle ).toHaveText( `Playwright Test Page #${ editor.postId }` );
+		await floatingElementTitle.hover();
+		const setAsEntireSite = page.locator( '.row-actions .set_as_entire_site' );
+		await expect( setAsEntireSite ).toBeVisible();
+		await setAsEntireSite.click();
+		await page.waitForURL( '/wp-admin/edit.php?post_type=e-floating-buttons' );
+
+		const context = await browser.newContext();
+		const newPage = await context.newPage();
+		await newPage.goto( 'https://example.com' );
+		await newPage.goto( '/' );
+		const floatingElement = newPage.locator( '.e-contact-buttons' );
+		await expect( floatingElement ).toBeVisible();
+
+		expect( await floatingElement.screenshot() ).toMatchSnapshot();
+
+		const button = floatingElement.locator( '.e-contact-buttons__chat-button' );
+		await expect( button ).toBeVisible();
+		await button.click();
+		const cta = floatingElement.locator( '.e-contact-buttons__send-button' );
+		await expect( cta ).toBeVisible();
+		const [ tabOpenedByButton ] = await Promise.all( [
+			context.waitForEvent( 'page' ),
+			// Perform the action that triggers the new tab, e.g., clicking a link
+			cta.click(),
+		] );
+
+		expect( tabOpenedByButton ).not.toBeNull();
+		await tabOpenedByButton.waitForLoadState( 'domcontentloaded' );
+
+		const [ secondtTabOpenedByButton ] = await Promise.all( [
+			context.waitForEvent( 'page' ),
+			// Perform the action that triggers the new tab, e.g., clicking a link
+			cta.click(),
+		] );
+
+		expect( secondtTabOpenedByButton ).not.toBeNull();
+		await secondtTabOpenedByButton.waitForLoadState( 'domcontentloaded' );
+
+		await tabOpenedByButton.close();
+		await secondtTabOpenedByButton.close();
+		await newPage.close();
+
+		await floatingElPage.goToFloatingButtonsPage();
+		const columnClickTracking = page.locator( '.wp-list-table tbody tr td.column-click_tracking' );
+		await expect( columnClickTracking ).toHaveText( '2' );
+	} );
+
+	test( 'Verify floating elements admin page behavior', async ( {
+		page,
+		apiRequests,
+	}, testInfo ) => {
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		await floatingElPage.goToFloatingButtonsEmptyPage();
+
+		const addNewButtonId = '#elementor-template-library-add-new';
+		const addNewButton = page.locator( addNewButtonId );
+
+		await test.step( 'Check that buttons and top bar exists', async () => {
+			await expect( addNewButton ).toBeVisible();
+			const topBar = page.locator( '#e-admin-top-bar-root' );
```

Comment:
I don't think we need to check of the Top Bar exists

Reply:
I think the Top Bar module itself will check if the Top Bar exists in their own tests. You should probably only check if your actions exist or not. WDYT?

Reply:
@louiswol94 since we are here on the page and the check takes milliseconds I would check it, don't see any downside.